docs/GLOSSARY.md

```diff
@@ -3,11 +3,17 @@
 This glossary defines common terms relevant for the PHP AI Client and related projects.
 
 * **Agent**: An autonomous system that can perceive its environment, make decisions, and take actions to achieve specific goals, often leveraging AI models.
+* **Candidate Count**: The number of different response options an LLM generates internally.
+* **Capability**: A specific skill, function, or type of task that an AI model can perform.
```

Comment:
I think it would be good to include an example:
```suggestion
* **Capability**: A specific skill, function, or type of task that an AI model can perform, e.g. text generation or image generation.
```